app/controllers/users/registrations_controller.rb

```diff
@@ -16,14 +16,35 @@ def after_inactive_sign_up_path_for(_resource)
     # POST /resource
 
     # GET /resource/edit
-    # def edit
-    #   super
-    # end
+     def edit
+       respond_to do |format|
+           format.js
+       end
+     end
 
     # PUT /resource
-    # def update
-    #   super
-    # end
+     def update
+       #super
+       self.resource = resource_class.to_adapter.get!(send(:"current_#{resource_name}").to_key)
```

Comment:
I think you can improve this method. I think resource is just a way to find the user, that can be done better with something like this https://www.railstutorial.org/book/updating_and_deleting_users#code-user_update_action_unsuccessful